src/Command/MeilisearchImportCommand.php

```diff
@@ -208,4 +200,24 @@ private function entitiesToIndex($indexes): array
 
         return array_unique($indexes->all(), SORT_REGULAR);
     }
+
+    /**
+     * @param string $prefixedIndexName
+     * @param string $entityClassName
+     * @param int    $batchSize
+     * @param int    $page
+     *
+     * @return array
+     */
+    private function getEntities($prefixedIndexName, $entityClassName, $batchSize, $page): array
+    {
+        $dataProvider = $this->searchService->getDataProvider($this->getIndexNameWithoutPrefix($prefixedIndexName));
```

Comment:
We could use ServiceLocator from Symfony instead of coupling this to the SearchService..